packages/daemons/confirmations.go

```diff
@@ -198,3 +204,18 @@ func IsReachable(host string, blockID int64, ch0 chan string, logger *log.Entry)
 		ch0 <- "0"
 	}
 }
+
+// NormalizeHostAddress get address. if port not defined returns combined string with ip and defaultPort
+func NormalizeHostAddress(address string, defaultPort int) (result string, err error) {
+	_, _, err = net.SplitHostPort(address)
+	if err == nil {
+		return address, nil
+	}
+
+	ip := net.ParseIP(address)
```

Comment:
adress could be host, not just ip

Reply:
if in first condition address contains port then address returns as correct value
else i check the correctness of the address and then add defaultPort